File: image_share_backFront-open/image_share_core/src/main/java/com/paperfly/imageShare/common/utils/EmptyUtil.java
package com.paperfly.imageShare.common.utils;

import java.util.Collection;
import java.util.Map;

public class EmptyUtil {
    public static boolean empty(String str) {
        return str == null || "".equals(str);
    }

    public static boolean empty(Map map) {
        return map == null || map.isEmpty();
    }

    public static boolean empty(Collection col){
        return col == null || col.isEmpty();
    }

    public static boolean empty(Object obj){
        return obj == null;
    }

    public static boolean empty(Object[] objs){
        return objs==null || objs.length==0;
    }
}

File: image_share_front-open/src/utils/EmptyUtil.js
export const EmptyUtil = {
    empty
}

function empty(data) {
    //判断对象
    if(!data && data !== 0){
        return true
    }
    //判断集合
    if (data.length || data.length === 0) {
        return data.length === 0
    }
    return false
}